sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -1414,4 +1415,37 @@ class DataFrame private[sql](
     val jrdd = rdd.map(EvaluatePython.rowToArray(_, fieldTypes)).toJavaRDD()
     SerDeUtil.javaToPython(jrdd)
   }
+
+  /////////////////////////////////////////////////////////////////////////////
+  // Statistic functions
+  /////////////////////////////////////////////////////////////////////////////
+
+  // scalastyle:off
+  object stat {
+  // scalastyle:on
+
+    /**
+     * Finding frequent items for columns, possibly with false positives. Using the algorithm
+     * described in `http://www.cs.umd.edu/~samir/498/karp.pdf`.
```

Comment:
should do proper citation rather than giving an url, since this url might disappear.